Add js-ipfs ProtoSchool tutorials (#2273)
ipfs/js-ipfs: examples/README.md

# `js-ipfs` Examples and Tutorials

> In this folder and on [ProtoSchool](https://proto.school), you can find a variety of examples to help you get started in using js-ipfs, in Node.js and in the Browser. Every example has a specific purpose and some of each incorporate a full tutorial that you can follow through, helping you expand your knowledge about IPFS and the Distributed Web in General.

Reviewer: ```suggestion
In this folder and on [ProtoSchool](https://proto.school), you can find a variety of examples to help you get started with js-ipfs, in Node.js and in the Browser. Every example has a specific purpose and some of them incorporate a full tutorial that you can follow through, helping you expand your knowledge about IPFS and the Distributed Web in general.
```
